Sources/TuistCore/ContentHashing/ContentHasher.swift

```diff
@@ -50,19 +50,26 @@ public final class ContentHasher: ContentHashing {
 
     public func hash(path filePath: AbsolutePath) throws -> String {
         if fileHandler.isFolder(filePath) {
-            let paths = try fileHandler.contentsOfDirectory(filePath)
-            let sortedPaths = paths.sorted(by: { $0 < $1 })
-            return try sortedPaths.map { try hash(path: $0) }.joined(separator: "-")
+            return try fileHandler
+                .contentsOfDirectory(filePath)
+                .filter { $0.basename.uppercased() != ".DS_STORE" }
```

Comment:
this is such a "nice" bug ahah  🏅 

nice work catching it, do we have other places were we hash directories? Maybe instead of filtering the `DS_STORE` we should filter for all files that are listed on the gitignore? I'd really try to prevent this bug from happening again because it must have been so hard to catch it